autoload/lsp/ui/vim/utils.vim

```diff
@@ -135,7 +135,7 @@ function! lsp#ui#vim#utils#diagnostics_to_loc_list(result) abort
     return l:list
 endfunction
 
-function! s:get_symbol_text_from_kind(server, kind) abort
+function! lsp#ui#vim#utils#get_symbol_text_from_kind(server, kind) abort
```

Comment:
Can we pefix `get_symbol_text_from_kind` with `_`. I would like to make this method internal for now.